mplc/scenario.py

```diff
@@ -17,42 +17,44 @@
 from sklearn.model_selection import train_test_split
 from sklearn.preprocessing import LabelEncoder
 
-from . import contributivity, constants, dataset as dataset_mod
-from . import multi_partner_learning
+from . import contributivity, constants
+from . import dataset as dataset_mod
```

Comment:
why dataset_mod ?

Reply:
shortcut for dataset module, to differ from real dataset.